src/browser/modules/Sidebar/ProjectsFilesScripts.tsx

```diff
@@ -206,14 +206,12 @@ function ProjectFilesScripts(props: ProjectFilesScripts): JSX.Element {
           isProjectFile: true
         })
       )
-    },
-    onExportScripts: Function.prototype,
-    onUpdateFolder: Function.prototype,
-    onRemoveFolder: Function.prototype
+    }
   }
 
   return (
     <>
+      {/* @ts-ignore */}
```

Comment:
what is being ignored?

Reply:
I've not yet fixed the project files to work with the new changes, so the ignore is there since they're broken! Will ping you for re-review when it's fixed O:)

Reply:
I've restructured how they work now, do try them out! (to do so add the preview-url as a graph app) I've speed up the loading of them since before we preloaded all the scripts just to list the names. And I moved them out to separate components, since they are different from favorites.